packages/core/src/hooks/mergeRefs.ts

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+type ReactRef<T> = React.Ref<T> | React.MutableRefObject<T>;
+
+export function mergeRefs<T>(
```

Comment:
perhaps use the `use...` nomenclature, or move to utils instead of hooks folder?? 🤔